stop-chromedriver.sh

```
@@ -0,0 +1,56 @@
+else
+    echo -e "${RED}⚠️  Some processes may still be running. Force killing...${NC}"
+    pkill -9 -f chromedriver
+    sleep 1
+    
+    FINAL_CHECK=$(pgrep -f chromedriver)
+    if [ -z "$FINAL_CHECK" ]; then
+        echo -e "${GREEN}✅ All ChromeDriver processes force-stopped${NC}"
+    else
+        echo -e "${RED}❌ Failed to stop some ChromeDriver processes${NC}"
+        echo "Remaining processes:"
+        ps aux | grep chromedriver | grep -v grep
```

Shellcheck (0.10.0)

[info] Consider using pgrep instead of grepping ps output.

(SC2009)